lib/postgresql.js

```diff
@@ -369,11 +375,7 @@ PostgreSQL.prototype.update =
     data = this.mapToDB(model, data);
     var props = this._categorizeProperties(model, data);
 
-    this.query(sql, generateQueryParams(data, props), function (err, result) {
-      if (callback) {
-        callback(err, result);
-      }
-    });
+    this.query(sql, generateQueryParams(data, props), callback);
```

Comment:
This is a breaking change, isn't it? Before your patch, one could call the method without providing any callback.

The correct fix:

```js
callback = callback || function(){};
this.query(sql, generateQueryParams(data, props), callback);
```